tidy3d/components/geometry.py

```diff
@@ -1562,13 +1562,22 @@ class PolySlab(Planar):
         units=RADIAN,
     )
 
+    reference_plane: PlanePosition = pydantic.Field(
+        "bottom",
+        title="Reference plane of polygon vertices",
+        description="The position of the plane where the supplied ``vertices`` are "
+        "defined. The ``vertices`` of the polygon "
+        "can be defined at the ``bottom``, ``middle``, or ``top`` of the PolySlab.",
```

Comment:
is `bottom` defined as the negative side of the axis? for example if axis=1 then bottom is y- side and top is y+ side? Maybe we can explicitly mention this.

Reply:
yeah, that's good to add. Please checkout the new commit.

Reply:
Looks good. Feel free to merge

Reply:
That is clearer. Edited.